client/jetpack-cloud/sections/agency-dashboard/hooks.tsx

```diff
@@ -46,11 +47,11 @@ export function useToggleActivateMonitor(
 	const { filter, search, currentPage } = useContext( SitesOverviewContext );
 	const queryKey = [ 'jetpack-agency-dashboard-sites', search, currentPage, filter ];
 
-	const toggleActivateMonitoring = useUpdateMonitorSettingsMutation( {
+	const toggleActivateMonitoring = useToggleActivateMonitorMutation( {
 		onMutate: async ( { siteId } ) => {
 			dispatch( setSiteMonitorStatus( siteId, 'loading' ) );
 		},
-		onSuccess: async ( data, { siteId } ) => {
+		onSuccess: async ( _data, { siteId, params } ) => {
```

Comment:
Is there any reason we prefixed this variable `_data` with an underscore?

Reply:
`data` here is not used anywhere.  

<img width="480" alt="Screenshot 2023-02-09 at 2 36 00 PM" src="https://user-images.githubusercontent.com/10586875/217766445-b9545f02-09a6-412b-8c70-c3bd26b1d930.png">

We can fix this warning by prefixing it with `_`

<img width="417" alt="Screenshot 2023-02-09 at 2 36 24 PM" src="https://user-images.githubusercontent.com/10586875/217767195-37b88045-34fe-49d6-b27b-d17271dee037.png">